README.md

```diff
@@ -2,56 +2,57 @@
+A numerical finite-difference approach is to be used for modeling heat transfer within the region of interest. Boundary conditions, material properties, heat generation rates, and other similar parameters will be based on data from established TRISO reactors.
 
 The program will produce a simple plot of the temperatures and fluxes within the fuel. Interactions with external material (such as coolant) will only be considered so far as they affect boundary conditions on the region of interest. Boundary conditions at the large pebble scale will be convective heat transfer to the coolant, or to the air in the event of coolant loss. Boundary conditions at the particle scale aren't quite as simple, and further evaluation is needed to determine what boundary condition would be appropriate.
 
 This project will aim to remain relatively simple in terms of modeling complexity. Only spherical geometries of concentric spheres (as in the fuel particles) will be modeled, though with varying thicknesses defined by the user. Transient analysis will focus on common transients such as reactor start up, shutdown, coolant loss or loss of forced convection. Transient analysis will only go so far as determining temperature and flux variations during the transient, and will not be able to account for damage to the fuel element as a result of temperature changes.
 
-Initially material properties will be considered constant with respect to position and temperature, with the intent to allow expansion to temperature dependent properties further along in the project. Heat generation at the particle scale will be uniform within the fueled region. At the pebble scale heat generation will be considered homogenous throughout the pebble. 
+Initially material properties will be considered constant with respect to position and temperature, with the intent to allow expansion to temperature dependent properties further along in the project. Heat generation at the particle scale will be uniform within the fueled region. At the pebble scale heat generation will be considered homogenous throughout the pebble.
+
+## Project Outline
 
 * Read input
 
 * Solve physics
 
   * Setup problem
-  
+
     * [Build mesh][i20]
     * [Define material properties][i21]
     * [Define time array][i22]
-    
+
   * loop over time steps
-  
-    * Define boundary conditions 
+
+    * [Define boundary conditions][i14]
     * Build matrix (Ax = B)
-    
+
       * [Build  matrix A][i17]
-      
+
         * Call material property (T,&alpha;)
-        * Call BC/IC
-        
+        * Call BC/IC (from [#14][i14])
+
       * [Build  matrix B][i18]
-      * Call material property (T,&alpha;)
-      * Call BC/IC
-      
+      * Call material property (T,&alpha;) from (from [#21][i21])
```

Comment:
FYI - you don't need to explicitly form the link for issues if you want the link text to just be `#21`.  If will automatically convert `#21` to be a link to issue 21

Reply:
We weren't sure if that feature worked for the README too, or just in comments. Thanks!

Reply:
I was completely wrong!!  What I said doesn't work... hmm...

Reply:
So the way it's currently written / the link I added using the same format is fine?